.travis.yml

```diff
@@ -12,6 +12,10 @@ dist: trusty
 # https://github.com/travis-ci/travis-ci/issues/9033
 sudo: required
 
+before_install:
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update          ; fi
```

Comment:
This seems to duplicate the `addons.homebrew.update = True` below

Reply:
Done, removed it, because it's taken care into the addons.homebrew.update = true.